src/components/bookDetails/BookDetails.tsx

```diff
@@ -66,35 +91,103 @@ const BookDetails = () => {
     setIsOpenEditModal(false)
   }
 
+  const handleRentBook = () => {
+    RentalServices.rentBook(bookDetails.Id)
+      .then(() => {
+        toast.success('Success rent book', {
+          position: toast.POSITION.TOP_RIGHT
```

Comment:
default position for message is top right, remove this property

Reply:
I fixed this.